basis/DoubleFFIScript.sml

```diff
@@ -0,0 +1,77 @@
+(*
+  Logical model of the Double state: TODO
```

Comment:
This TODO should be removed before this gets merged.

Reply:
Done.